funnel/views/proposal.py

```diff
@@ -88,13 +90,12 @@ def new_proposal(self):
             )
             return redirect(proposal.url_for(), code=303)
 
-        return render_form(
-            form=form,
-            title=_("Make a submission"),
-            submit=_("Submit"),
-            message=markdown_message,
-            cancel_url=self.obj.url_for(),
-        )
+        return {
+            'title': "New submission",
+            'form': form,
+            'project': self.obj,
+            'proposal': proposal if proposal else '',
```

Comment:
If the default changes to `None` above, this can be simply `'proposal': proposal`